internal/service/tech_server.go

```diff
@@ -0,0 +1,20 @@
+package service
+
+import (
+	"github.com/etf1/kafka-mongo-watcher/internal/server"
+)
+
+func (container *Container) GetTechServer() *server.TechServer {
+	if container.techServer == nil {
+		container.techServer = server.NewTechServer(
+			container.GetLogger(),
+			container.Cfg.TechServer.HTTPAddr,
```

Comment:
we can use directly `container.Cfg.HTTPAddr` instead

Reply:
We have 4-5 configuration variables related to HTTP tech server (this one, timeouts, ...) os I prefer to keep them in a separated section, I don't think it's a problem